pkg/minikube/constants/constants.go

```diff
@@ -38,6 +38,9 @@ const MiniShiftEnvPrefix = "MINISHIFT"
 // MiniShiftHomeEnv is the environment variable used to change the Minishift home directory
 const MiniShiftHomeEnv = "MINISHIFT_HOME"
 
+const MinSupportedOpenshiftTag = ">1.3.1"
```

Comment:
why not '>='? We started with 1.3.1, right?

Right now I get:

    minishift start --openshift-version v1.3.1
    F0215 10:31:04.744867    4867 start.go:337] v1.3.1 is not supported with minishift